src/Sensors/IMU/BNO055.cpp

```diff
@@ -21,9 +21,11 @@ namespace mmfs
 
     void BNO055::read()
     {
-        measuredAcc = convertIMUtoMMFS(bno.getVector(Adafruit_BNO055::VECTOR_LINEARACCEL));
-        measuredGyro = convertIMUtoMMFS(bno.getVector(Adafruit_BNO055::VECTOR_GYROSCOPE));
-        measuredMag = convertIMUtoMMFS(bno.getVector(Adafruit_BNO055::VECTOR_MAGNETOMETER));
+        accelerationVec = convertIMUtoMMFS(bno.getVector(Adafruit_BNO055::VECTOR_LINEARACCEL));
```

Comment:
Will these already be in the inertial frame from the BNO055? Should we endeavor to make sure they aren't?

Reply:
No. The bno returns it's measurements in the body frame AND it returns an orientation quaternion. It is up to the user of the bno to do the transformation from the body to inertial using that orientation. 

This is how we should and I did set up our IMU class. We just add a helper function that does that transformation for you if you want.